### PR TITLE
Update NOTICE.txt

### PR DESCRIPTION
**Pull request by amyblais**
To match with https://github.com/mattermost/mattermost-webapp/blob/release-5.26/package.json.

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -392,6 +347,10 @@ THE SOFTWARE.
 
 ---
 
+## dynamic-virtualized-list
```
Comment (amyblais) [inline NOTICE.txt:350]: Unsure if it's https://github.com/mattermost/dynamic-virtualized-list or https://github.com/dwqs/react-virtual-list

**Comment by amyblais** [comment]
@grundleborg Kind reminder to review for v5.26,

File: NOTICE.txt (hunk 1 above)
Comment (grundleborg) [reply NOTICE.txt:350]: It's this: https://github.com/mattermost/dynamic-virtualized-list

You can find the package names in `package.json`. If they are URLs then that tells you which one it is (as in this case). If they are just names, then you can search on https://npmjs.com to find the package by it's name, and then the link to Github will be there.

Comment (amyblais) [reply NOTICE.txt:350]: @grundleborg Thanks, added

**Comment by grundleborg** [comment]
@amyblais you've got conflicts to resolve by the looks of it.

**Pull request merged**